### ISSUE TITLE
Failed to import games . . . Error converting value {null}

### ISSUE THREAD

**Issue by jinoru** [XenorPLxx/playnite-library-psn#3]
Playstation library does not import with the following error:
```
Failed to import games from Playstation.
Error converting value {null} to type 'System.DateTime'. Path 'data.gameLibraryTitlesREtrieve.games[77].lastPlayedDateTime', line 1, position 38078
```
![image](https://user-images.githubusercontent.com/4512892/140636501-e09351bd-00a5-4116-871e-0e72425c8bb6.png)
This occurs each time Playnite updates the game library for Playstation, regardless of the add-on settings.

Diagnostics ID
c922ffa4-a561-49bc-9d40-be36f77f6278

**Comment by XenorPLxx** [comment]
@jinoru I've released 2.12 version that should fix this problem. Let me know if it worked.

**Comment by jinoru** [comment]
That worked. Thanks
